### ISSUE TITLE
Incorrect data32/data64 type in clap_audio_buffer

### ISSUE THREAD

**Issue by ilmai** [glowcoil/clap-sys#29]
In audio-buffer.h, data32 and data64 have types `float**` and `double**`, respectively. In clap-sys, these translate to `*const *const f32` and `*const *const f64`. They should be `*mut *mut` instead to match the C API.

**Comment by micahrj** [comment]
You're right, and it would be good to fix this going forward. I'm not sure why the pointers were `*const` in the first place.

It is worth pointing out that this makes no difference in terms of either FFI correctness or Rust safety, so there's no risk of this causing issues for existing code and the fix would be purely for clarity's sake.

**Comment by micahrj** [comment]
Fixed in #36.
